src/unix/linux_like/emscripten/mod.rs

```diff
@@ -867,22 +863,28 @@ pub const SHM_UNLOCK: ::c_int = 12;
 pub const SHM_HUGETLB: ::c_int = 0o4000;
 pub const SHM_NORESERVE: ::c_int = 0o10000;
 
-pub const QFMT_VFS_OLD: ::c_int = 1;
-pub const QFMT_VFS_V0: ::c_int = 2;
+// No quota.h
+// https://github.com/emscripten-core/emscripten/pull/17704
+/*pub const QFMT_VFS_OLD: ::c_int = 1;
+pub const QFMT_VFS_V0: ::c_int = 2;*/
```

Comment:
It is awesome to have the emscripten PRs linked, but could you put them all into the commit message and delete the commented-out code? Since these are things we want to keep in history but it isn't relevant to the code anymore.

Reply:
Good idea! Split into commit 476bf225658143f5689fa2986e6fb491c8081eaf and 99f8c8751f94d6149e55c6d238f310303694ca4c.